source/administration/devops-command-center.rst

```diff
@@ -9,159 +9,95 @@ DevOps Command Center
-
-- Name
-- Status (**Ongoing** or **Ended**)
-- Start Date
-- End Date, if ended, otherwise **--**
-- Commander
-
-Viewing incident details
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To end an active incident, browse to the incident channel and use one of the following steps:
 
-To view details of active incidents, select **! Incidents** from the plugin menu, or **!** in the channel header to open the RHS panel.
+- The slash command */incident end*.
+- The **End Incident** button in the RHS.
 
-Listed incidents include the following information:
-
-- The current commander’s profile picture and username.
-- Zero or more checklist items.
-- The channels associated with the incident.
-- A button to end the incident.
+Browsing incidents
+~~~~~~~~~~~~~~~~~~
 
-If the incident is active, but in a private channel, you won't be able to see the incident channel name or select **End Incident**. However, the commander name and incident checklist are displayed. If you're a participant in an incident channel, you can modify incident details from within that channel.
+1. Navigate to **Main Menu > Incidents & Playbooks**.
+2. Select **Incidents** in the top menu to view the incidents in the current team.
 
-You can view all incidents in the incident Backstage via **Main Menu > Incidents & Playbooks** and then select an incident to view the following details:
+Only public incidents and incidents in which the active user is a member are displayed. System administrators have access to all incidents, whether or not they are members.
 
-- Incident name
-- A link icon to open the corresponding incident channel
-- Status (**Ongoing** or **Ended**)
-- The commander, including profile picture and username
-- A prompt to `Export the Incident Channel <#exporting-channels>`_
-- The **Duration** widget displays the duration of the incident. While the incident is ongoing, the end time is displayed as **Ongoing**. When the incident has ended, it shows the end time (in the user's timezone).
-- The **Members Involved** widget indicates the total number of users that participated in the channel, either by posting a message, being assigned as commander, or interacting with a checklist. This number is not affected by users leaving the channel, or users joining the channel but not participating.
-- The total number of messages displayed includes messages posted by both users and bots (including the incident response bot). It does not include system or ephemeral messages.
-- A graph depicting when each checklist item was completed.
+Click on an incident to view additional metadata such as duration, number of involved members, number of messages posted to the channel, and a timeline of when steps were resolved. Optionally filter the list of incidents to include only ongoing incidents or the incident's commander.
```

Comment:
I did not understand the last part of this line, "Optionally filter the list of incidents to include only ongoing incidents or the incident's commander." Do you mean filtering by commander?